Index: lib/HighlightHashLink.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/HighlightHashLink.js b/lib/HighlightHashLink.js
--- a/lib/HighlightHashLink.js	
+++ b/lib/HighlightHashLink.js	
@@ -10,7 +10,7 @@
 /**
  * This is a utility function to highlight links in a hash menu.
  */
-import { prefixClass } from '../utils/prefixClass'
+import prefixClass from '../utils/prefixClass'
 
 function highlightActiveLinks (linkSelector) {
   const navLinks = document.querySelectorAll(linkSelector)
Index: lib/validation/utils/helpers.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/validation/utils/helpers.js b/lib/validation/utils/helpers.js
--- a/lib/validation/utils/helpers.js	
+++ b/lib/validation/utils/helpers.js	
@@ -8,7 +8,7 @@
  */
 
 import getScrollTop from '../../../utils/getScrollTop'
-import { prefixClass } from '../../../utils/prefixClass'
+import prefixClass from '../../../utils/prefixClass'
 
 function getAllInputsArray (form) {
   return Array.from(form.querySelectorAll('[pattern], [required], input[type="email"], .tiptap__input--hidden.is-required, .multiselect.is-required, fieldset.is-required, [minlength], [maxlength], [data-dp-validate-maxlength], [data-dp-validate-should-equal]'))
Index: mixins/tableSelectAllItems.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mixins/tableSelectAllItems.js b/mixins/tableSelectAllItems.js
--- a/mixins/tableSelectAllItems.js	
+++ b/mixins/tableSelectAllItems.js	
@@ -28,7 +28,7 @@
  *   @items-toggled="handleToggleItem" />
  */
 
-import { uniqueArrayByObjectKey } from '../index'
+import uniqueArrayByObjectKey from '../utils/uniqueArrayByObjectKey'
 
 export default {
   data () {
Index: lib/ToggleAnything.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/ToggleAnything.js b/lib/ToggleAnything.js
--- a/lib/ToggleAnything.js	
+++ b/lib/ToggleAnything.js	
@@ -8,7 +8,7 @@
  */
 
 import getAnimationEventName from '../utils/getAnimationEventName'
-import { prefixClass } from '../utils/prefixClass'
+import prefixClass from '../utils/prefixClass'
 /**
  *  ToggleAnything
  *  Adds toggle behavior to elements
Index: mixins/dpValidateMixin.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mixins/dpValidateMixin.js b/mixins/dpValidateMixin.js
--- a/mixins/dpValidateMixin.js	
+++ b/mixins/dpValidateMixin.js	
@@ -13,7 +13,9 @@
  * Up to date documentation and usage examples can be found here:
  * https://yaits.demos-deutschland.de/w/demosplan/frontend-documentation/frontend-validierung/
  */
-import { assignHandlersForInputs, hasOwnProp, validateForm } from '../index'
+import { assignHandlersForInputs } from '../lib/validation/utils/assignHandlersForInputs'
+import hasOwnProp from '../utils/hasOwnProp'
+import validateForm from '../lib/validation/utils/validateForm'
 import { errorClass, scrollToVisibleElement } from '../lib/validation/utils/helpers'
 
 export default {
Index: lib/TableWrapper.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/TableWrapper.js b/lib/TableWrapper.js
--- a/lib/TableWrapper.js	
+++ b/lib/TableWrapper.js	
@@ -19,7 +19,7 @@
  *  <element data-table-wrapper="overflow-x-auto">
  *
  */
-import { prefixClass } from '../utils/prefixClass'
+import prefixClass from '../utils/prefixClass'
 
 const wrapElement = function (element, wrapper) {
   element.parentNode.insertBefore(wrapper, element)
Index: lib/ActionMenu.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/ActionMenu.js b/lib/ActionMenu.js
--- a/lib/ActionMenu.js	
+++ b/lib/ActionMenu.js	
@@ -40,7 +40,7 @@
  *  To enable highlighting of current item, drop data-actionmenu-current on an item
  *
  */
-import { prefixClass } from '../utils/prefixClass'
+import prefixClass from '../utils/prefixClass'
 
 class ActionMenu {
   constructor (actionMenuElement) {
Index: lib/Tooltips.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/Tooltips.js b/lib/Tooltips.js
--- a/lib/Tooltips.js	
+++ b/lib/Tooltips.js	
@@ -7,7 +7,7 @@
  * All rights reserved
  */
 
-import { prefixClass } from '../utils/prefixClass'
+import prefixClass from '../utils/prefixClass'
 /*
  *
  *JS-Solution to display Tooltips.
Index: utils/hasOwnProp.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/utils/hasOwnProp.js b/utils/hasOwnProp.js
--- a/utils/hasOwnProp.js	
+++ b/utils/hasOwnProp.js	
@@ -13,7 +13,7 @@
  * @param prop
  * @return {boolean}
  */
-export default function hasOwnProp (obj, prop) {
+export default function (obj, prop) {
   if (typeof obj !== 'object') {
     console.warn('Cannot check for property on a non-object, got type: ' + typeof obj)
 
Index: lib/DpApi.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/DpApi.js b/lib/DpApi.js
--- a/lib/DpApi.js	
+++ b/lib/DpApi.js	
@@ -8,7 +8,7 @@
  */
 
 import axios from 'axios'
-import { hasOwnProp } from '../index'
+import hasOwnProp from '../utils/hasOwnProp'
 import { stringify } from 'qs'
 import { v4 as uuid } from 'uuid'
 
Index: lib/Tabs.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/Tabs.js b/lib/Tabs.js
--- a/lib/Tabs.js	
+++ b/lib/Tabs.js	
@@ -7,7 +7,7 @@
  * All rights reserved
  */
 
-import { prefixClass } from '../utils/prefixClass'
+import prefixClass from '../utils/prefixClass'
 
 /**
  *  Tabs - add behavior to tabbed elements.
Index: lib/validation/dpValidateMultiselectDirective.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/validation/dpValidateMultiselectDirective.js b/lib/validation/dpValidateMultiselectDirective.js
--- a/lib/validation/dpValidateMultiselectDirective.js	
+++ b/lib/validation/dpValidateMultiselectDirective.js	
@@ -7,7 +7,7 @@
  * All rights reserved
  */
 
-import { validateMultiselect } from '../../index'
+import validateMultiselect from './utils/validateMultiselect'
 
 const dpValidateMultiselectDirective = {
   inserted (el, binding, vnode) {
Index: index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.js b/index.js
--- a/index.js	
+++ b/index.js	
@@ -30,6 +30,8 @@
 import hasOwnProp from './utils/hasOwnProp'
 import { highlightActiveLinks } from './lib/HighlightHashLink'
 import initGlobalEventListener from './lib/GlobalEventListener'
+import initToggleSideMenu from './lib/ToggleSideMenu'
+import initUserbox from './lib/touchFriendlyUserbox'
 import MatchMedia from './lib/MatchMedia'
 import NotificationStoreAdapter from './lib/NotificationStoreAdapter'
 import Pager from './lib/Pager'
@@ -42,9 +44,7 @@
 import Tabs from './lib/Tabs'
 import throttle from './utils/throttle'
 import ToggleAnything from './lib/ToggleAnything'
-import ToggleSideMenu from './lib/ToggleSideMenu'
 import Tooltips from './lib/Tooltips'
-import touchFriendlyUserbox from './lib/touchFriendlyUserbox'
 import uniqueArrayByObjectKey from './utils/uniqueArrayByObjectKey'
 
 export {
@@ -69,16 +69,18 @@
   getFileInfo,
   getFileTypes,
   handleResponseMessages,
+  hasAllPermissions,
+  hasAnyPermissions,
+  hasOwnProp,
+  hasPermission,
   highlightActiveLinks,
   initGlobalEventListener,
+  initToggleSideMenu,
+  initUserbox,
   isActiveFullScreen,
   getAnimationEventName,
   getCssVariable,
   getScrollTop,
-  hasAllPermissions,
-  hasAnyPermissions,
-  hasOwnProp,
-  hasPermission,
   makeFormPost,
   MatchMedia,
   mimeTypes,
@@ -92,12 +94,10 @@
   TableWrapper,
   Tabs,
   ToggleAnything,
-  ToggleSideMenu,
   Tooltips,
   throttle,
   toDate,
   toggleFullscreen,
-  touchFriendlyUserbox,
   unbindFullScreenChange,
   uniqueArrayByObjectKey
 }
